### DISCUSSION TITLE
Is this implementation FIDO CERTIFIED or FIDO CONFORMANT ?

### DISCUSSION THREAD

**Discussion by sheeri185adidam** [duo-labs/py_webauthn#231] [Q&A] [answered]
Hey guys,

I am in the initial phase of planning to implement a backend service that provides endpoints for web authentication. I wanted to ask if this library is `FIDO CERTIFIED` or `FIDO CONFORMANT` ?

**Comment by MasterKale** [comment, marked as answer]
No, I've not going through FIDO Conformance with this library yet. More for lack of time than anything unfortunately. This may change in a few months though 🤔
